test/box-luatest/gh_7743_term_initial_cfg_snap_test.lua

```diff
@@ -20,7 +20,6 @@ g.test_sigterm_during_initial_snapshot = function()
             -- uses usleep() which is a pthread cancellation point.
             TARANTOOL_RUN_BEFORE_BOX_CFG = [[
                 box.ctl.set_on_shutdown_timeout(1000)
-                box.error.injection.set('ERRINJ_MAIN_MAKE_FILE_ON_RETURN', true)
```

Comment:
Why? Does it stop working?

Reply:
Yeah, in this test we terminate server during execution of init script. Before this patch server exits with 0 exit code but after the patch server will panic due to `FiberIsCancelled` exception during init script execution.

Dropped `ERRINJ_MAIN_MAKE_FILE_ON_RETURN` as discussed:

<details>
<summary>diff</summary>

```diff
diff --git a/src/lib/core/errinj.h b/src/lib/core/errinj.h
index 89d81a606..12e5412fd 100644
--- a/src/lib/core/errinj.h
+++ b/src/lib/core/errinj.h
@@ -104,7 +104,6 @@ struct errinj {
 	_(ERRINJ_IPROTO_TX_DELAY, ERRINJ_BOOL, {.bparam = false}) \
 	_(ERRINJ_IPROTO_WRITE_ERROR_DELAY, ERRINJ_BOOL, {.bparam = false})\
 	_(ERRINJ_LOG_ROTATE, ERRINJ_BOOL, {.bparam = false}) \
-	_(ERRINJ_MAIN_MAKE_FILE_ON_RETURN, ERRINJ_BOOL, {.bparam = false}) \
 	_(ERRINJ_MEMTX_DELAY_GC, ERRINJ_BOOL, {.bparam = false}) \
 	_(ERRINJ_NETBOX_DISABLE_ID, ERRINJ_BOOL, {.bparam = false}) \
 	_(ERRINJ_NETBOX_FLIP_FEATURE, ERRINJ_INT, {.iparam = -1}) \
diff --git a/src/main.cc b/src/main.cc
index 42f3a68a2..2bdf67593 100644
--- a/src/main.cc
+++ b/src/main.cc
@@ -1109,14 +1109,5 @@ main(int argc, char **argv)
 	free((void *)instance.name);
 	free((void *)instance.config);
 	tarantool_free();
-	ERROR_INJECT(ERRINJ_MAIN_MAKE_FILE_ON_RETURN, do {
-		int fd = open("tt_exit_file.txt.inprogress",
-			      O_WRONLY | O_CREAT | O_TRUNC, -1);
-		if (fd < 0)
-			break;
-		dprintf(fd, "ExitCode: %d\n", exit_code);
-		close(fd);
-		rename("tt_exit_file.txt.inprogress", "tt_exit_file.txt");
-	} while (false));
 	return exit_code;
 }
diff --git a/test/box/errinj.result b/test/box/errinj.result
index 8c4c8150f..979031600 100644
--- a/test/box/errinj.result
+++ b/test/box/errinj.result
@@ -77,7 +77,6 @@ evals
   - ERRINJ_IPROTO_TX_DELAY: false
   - ERRINJ_IPROTO_WRITE_ERROR_DELAY: false
   - ERRINJ_LOG_ROTATE: false
-  - ERRINJ_MAIN_MAKE_FILE_ON_RETURN: false
   - ERRINJ_MEMTX_DELAY_GC: false
   - ERRINJ_NETBOX_DISABLE_ID: false
   - ERRINJ_NETBOX_FLIP_FEATURE: -1

```

</details>